fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -27,6 +26,9 @@ import { Question } from '../objects';
 
 const DEFAULT_ELECTION_DURATION = 3600;
 
+// for now only plurality voting is supported
```

Comment:
You should maybe add a date and your GitHub name there to indicate when it was written, like this :
```suggestion
// for now only plurality voting is supported (2022-03-16, Tyratox)
```